consensus/spos/extraSigners/sovereignSubRoundEndExtraSigner.go

```diff
@@ -108,9 +115,12 @@ func (sr *sovereignSubRoundEndOutGoingTxData) SignAndSetLeaderSignature(header d
 		return nil
 	}
 
-	leaderMsgToSign := append(
-		outGoingMb.GetOutGoingOperationsHash(),
-		outGoingMb.GetAggregatedSignatureOutGoingOperations()...)
+	leaderMsgToSign := outGoingMb.GetOutGoingOperationsHash()
+
+	// In consensus v2 leader will only sign the outgoing op hash
+	if !sr.enableEpochsHandler.IsFlagEnabled(common.AndromedaFlag) {
+		leaderMsgToSign = append(leaderMsgToSign, outGoingMb.GetAggregatedSignatureOutGoingOperations()...)
```

Comment:
andromeda is activated by default.

Reply:
Yes, agree, it will be done this way, as mentioned in previous PRs, this is for viebechain upgrade